src/constants.js

```diff
@@ -45,12 +36,19 @@ export const STORAGE_KEYS = {
   SELECTED_ADDRESS: "userSelectedAddress",
 };
 
-export const SUGGESTED_TAGS = {
-  fruits: "Fruits",
-  vegetables: "Vegetables",
-  meat: "Meat",
-  eggs_dairy: "Eggs & Dairy",
-  pulses_grains: "Pulses & Grains",
-  medicines: "Medicines",
-  prepared_food: "Prepared Food",
-};
+export const SUGGESTED_TAGS = [
+  "fruits",
+  "vegetables",
+  "meat",
+  "eggs_dairy",
+  "pulses_grains",
+  "medicines",
+  "prepared_food",
+];
+
+export const SAFETY_CHECKS_LIST = [
+  "long_wait",
+  "masks_required",
+  "adequate_social_distancing",
+  "home_delivery",
+];
```

Comment:
moved away from object to array since, `Object.keys` was not maintaining the item order inside render

![order](https://user-images.githubusercontent.com/5427715/80044113-d1d4c600-8503-11ea-877f-68fceae8e436.gif)